### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by aliakhraisha**
# Setting up the "hello Microverse" project
- Edit  Readme file 
- Create .gitignore file 
- Configure lint 
- Add text to h1 tag
- Style h1 text in the CSS file 
- Fix linter errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,55 +2,31 @@
...
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
+No  setup required
```
Comment (ha-manel) [inline README.md]: - [ ] Kindly fill out this section. It's supposed to describe how to setup the project locally so you can write instructions on how to clone the project and run it in the browser etc...

File: README.md [hunk 2]
```diff
@@ -2,55 +2,31 @@
...
-- Major languages
-- Frameworks
-- Technologies used
+- HTML, CSS
+- VSC, github
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://www.youtube.com/watch?v=UTga0ARWu2g&t=252s)
```
Comment (ha-manel) [inline README.md]: - [ ] Kindly remove this section since you don't have a live demo of the project 👍

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @aliakhraisha 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 

## Required Changes:
- [ ] Kindly change the pull request's title. It should have a descriptive title telling you what changes are inside.
- [ ] Kindly change the pull request's description. It should have a summary describing introduced changes in more detail.
- [ ] From now on please select the branch's name carefully. It should have the name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section').

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (aliakhraisha) [reply README.md]: done

Comment (aliakhraisha) [reply README.md]: done

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅
## Hi @aliakhraisha 👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**

 

## Highlights:
* Well written PR description ✔️ 
* Proper use of GitHub flow ✔️  
* All tests passed ✔️ 

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**